launch_ros/launch_ros/utilities/to_parameters_list.py

```diff
@@ -28,25 +29,69 @@
 from ..parameters_type import EvaluatedParameters
 
 
+def __format_dict(dictionary, keys, formatted_dict):
```

Comment:
@jacobperron should we try to come up with a better name?

Reply:
I renamed this to `__normalized_parameters_dict` (9de0767)
Please suggest a different name if you think of a better one.